lecture_02/assignment_01/ruxandra-chiujdea/assignment_01.py

```diff
@@ -0,0 +1,37 @@
+"""Assignment 01: Project box to xy-plane
+"""
+from compas.artists import Artist
+from compas.datastructures import Mesh
+from compas.geometry import Box
+from compas.geometry import Frame
+from compas.geometry import Plane
+from compas.geometry import Projection
+
+# Define a Frame, which is not in the origin and a bit tilted to the world frame
+frame = Frame([20, 0, 10], [-0.45, 0.1, 0.3], [1, 0, 0])
+plane = Plane([0,0,0],[0,0,1])
+direction = [-0.45,0,-1]
+perspective = [6,8,20]
```

Comment:
Code style: add whitespace after commas:

```suggestion
plane = Plane([0, 0, 0], [0, 0, 1])
direction = [-0.45, 0, -1]
perspective = [6, 8, 20]
```